integrationtest/src/test/java/org/mapstruct/itest/tests/LombokBuilderTest.java

```diff
@@ -27,6 +27,12 @@
  */
 @RunWith( ProcessorSuiteRunner.class )
 @ProcessorSuite( baseDir = "lombokBuilderTest",
-    processorTypes = ProcessorSuite.ProcessorType.ALL_WITHOUT_PROCESSOR_PLUGIN)
```

Comment:
why not eclipse?

Reply:
For eclipse we need to use the lombok agent, or somehing like that. Lombok goes deep into the internals of the compilers and does not work out of the box with Eclipse

Reply:
sigh.. I know.. you have to run a special tool that installs an additional jar on the eclipse class path..